src/main/resources/application-local.yml

```diff
@@ -17,6 +17,9 @@ spring:
         jwt:
           public-key-location: classpath:local-public-key.pub
 
+  jpa:
+    show-sql: true
```

Comment:
sure

Reply:
Sorry meant to remove that one, will leave it.